addons/workspace-dots/addon.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "Editor workspace dots theme",
+  "description": "Allows the customisation of the style of the editor workspace dots.",
+  "tags": ["editor", "codeEditor", "theme"],
+  "versionAdded": "1.39.0",
+  "userscripts": [{ "url": "userscript.js", "matches": ["projects"] }],
+  "settings": [
+    {
+      "name": "Dots theme",
+      "id": "theme",
+      "type": "select",
+      "default": "dots",
+      "potentialValues": [
+        { "id": "dots", "name": "Default dots" },
+        { "id": "crosshairs", "name": "Crosshairs" },
+        { "id": "lines", "name": "Solid lines" },
+        { "id": "vertical", "name": "Vertical Lines" },
+        { "id": "horizontal", "name": "Horizontal Lines" },
+        { "id": "none", "name": "None" }
+      ]
+    },
+    {
+      "name": "Use a custom spacing value",
```

Comment:
whats the point of this setting?

Reply:
That disables the custom spacing setting and uses the default spacing provided by the snapping addon